.github/workflows/ci.yml

```diff
@@ -0,0 +1,61 @@
+name: Scala Merge CI
+
+on:
+  push:
+    branches: ['2.*.x']
+  pull_request:          ## TODO remove this line before merging
+
+defaults:
+  run:
+    shell: bash
+
+jobs:
+  build_and_test:
+    name: Windows
+    runs-on: windows-latest
+    strategy:
+      fail-fast: false
+    steps:
+      - run: git config --global core.autocrlf false
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      # Note that we don't use olafurpg/setup-scala; it wouldn't buy us anything
+      # over setup-java. (We don't want csbt or xsbt; we prefer the standard
+      # sbt launch script, which comes preinstalled on Windows (and Ubuntu).)
+      - name: Setup Java
+        uses: actions/setup-java@v2
+        with:
+          distribution: adopt
+          java-version: 8
+
+      - name: Cache
+        uses: actions/cache@v2
+        with:
+          path: |
+            ~/.sbt
+            ~/.ivy2/cache
+            ~/.cache/coursier
+          key: ${{ runner.os }}-sbt-cache-v2-${{ hashFiles('**/*.sbt') }}-${{ hashFiles('project/build.properties') }}
+
+      - name: Setup
+        run: |
+          source scripts/common
+          java -version
+          javac -version
+          generateRepositoriesConfig
+          # Pass these environment vars to subsequent steps
+          echo "SBT=sbt -Dsbt.override.build.repos=true -Dsbt.repository.config=${sbtRepositoryConfig}" >> $GITHUB_ENV
+          echo "COURSIER_HOME=$HOME/.coursier" >> "$GITHUB_ENV"
+          echo "COURSIER_CACHE=$HOME/.cache/coursier/v1" >> "$GITHUB_ENV"
+
+      - name: Build
+        run: |
+          source scripts/common
+          $SBT -warn setupPublishCore generateBuildCharacterPropertiesFile publishLocal
+
+      - name: Test
+        run: |
+          source scripts/common
+          parseScalaProperties buildcharacter.properties
+          $SBT -Dstarr.version=$maven_version_number -warn setupValidateTest testAll
```

Comment:
why `-warn`?

Reply:
beats me, but `.travis.yml` has it as well

Reply:
Ah, on 2.12, but not anymore on 2.13.

Reply:
ah, good catch. I'll remove it